adapters-1.6/src/test/java/com/flipkart/fdp/ml/adapter/VectorBinarizerBridgeTest.java

```diff
@@ -0,0 +1,146 @@
+package com.flipkart.fdp.ml.adapter;
+
+import com.flipkart.fdp.ml.export.ModelExporter;
+import com.flipkart.fdp.ml.importer.ModelImporter;
+import com.flipkart.fdp.ml.transformer.Transformer;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.ml.feature.VectorBinarizer;
+import org.apache.spark.mllib.linalg.DenseVector;
+import org.apache.spark.mllib.linalg.SparseVector;
+import org.apache.spark.mllib.linalg.VectorUDT;
+import org.apache.spark.sql.DataFrame;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.RowFactory;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.Metadata;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ * Created by karan.verma on 09/11/16.
+ */
+public class VectorBinarizerBridgeTest extends SparkTestBase{
+
+    @Test(expected=IllegalArgumentException.class)
+    public void testVectorBinarizerNegativeThresholdValue() {
+        // prepare data
+
+        JavaRDD<Row> jrdd = sc.parallelize(Arrays.asList(
```

Comment:
Just for clarity, can you remove the bits where you create the RDD, since the test doesn't require it here.

I think it will make the code easier to read.